jarbas/README.md

```diff
@@ -84,6 +95,8 @@ This request will list:
 
 Also you can pass more than one value per field (e.g. `document_id=111111,222222`).
 
+Search field
+
```

Comment:
Are those two lines needed? I felt they ended up kind of meaningless. Can you check this please?